### ISSUE TITLE
Milestone 3 Feedback

### ISSUE THREAD

**Issue by tonyliang19** [miniatureseal/dsci-310-group-10-GCC#158]
# Milestone 3 Feedback Group 10

## Anaylysis abstracted to four or more scripts, which take command line arguments and save analysis artifacts (figures and data for tables)

- [x] scripts except `01_data_loading.R` do not use command line argument and those naming paths for files to be written to csv / or saveing to figs could be abstracted to command line argument **-4**

## Report written in literate code document code, which primarily functions to source analysis artifacts and format document

- [x] Extra code for confusion matrix. Again you had enough time to ask instructors team, deadline was again pushed to day after of original due date. You could ve plot it and ggsave it, then just load inside the report. **-2**

## Project file and directory structure organization

- [x] `analysis.Rmd` could be in the `results` directory as well and not in the top-level **-1**

Side Note:

Consider using a `reference.bib` and put it together within the same directory as your `analysis.Rmd`

## Analysis can be run reproducibly using documentation provided in README

~~- [ ] No such target for `make test` exists **-2**~~
- [x] Did not use auto referencing and makes it hard to follow which figure/table referring to table `2.2.1 and 2.2.2` **-2**
- [x] Check your numbers, details matters, you could have use inline R code to show that **-1**, see below:

![image](https://user-images.githubusercontent.com/83138402/230212549-8dddd0c8-fa45-473b-82a5-3957b562e64a.png)

## Address Feedback from Milestone 1 and 2

Up to 50 % of Milestone 1 + Milestone 2 points, if feedback and comments are adequately addressed

### [Milestone 2 Feedback](https://github.com/miniatureseal/dsci-310-group-10-GCC/issues/103)

None addressed by April 05

Check below:

![image](https://user-images.githubusercontent.com/83138402/230212578-5567fe12-6427-4241-84dd-0fbf55c3be20.png)
![image](https://user-images.githubusercontent.com/83138402/230212599-1baa7a47-bc93-4352-82eb-ad4e489d9ef5.png)

### [Milestone 1 Feedback](https://github.com/miniatureseal/dsci-310-group-10-GCC/issues/53)

- `README.md` **50% of 1 = 0.5**
- Analysis **50% of 6 = 3**

Total = $0.5 + 3 = 3.5$ points

**Comment by tonyliang19** [comment]
Removed the task for `make test`, it is my mistake here. Im sorry about that, and your marks will be fixed as well on Canvas

**Comment by miniatureseal** [comment]
Above mentioned issues will be addressed with #171 #174 #175 #178 #179.
We will also update this ticket accordingly.

**Comment by miniatureseal** [comment]
All tasks done!
